spec/data/boolean.yaml

```diff
@@ -1,5 +1,6 @@
 form:
   title: "Bool test"
+  id: 7e6f5289d887d8afb01b17e044e9c148f4b8d4e5
   version: "1"
```

Comment:
I think we want to maintain major/minor versions of exams...at least for the purposes of breaking/non-breaking changes.